config/bidderinfo.go

```diff
@@ -356,6 +358,44 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) > 0 {
+		if parentBidder, ok := infos[bidder.AliasOf]; ok {
+			if parentBidder.Capabilities == nil && bidder.Capabilities != nil {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
```

Comment:
how about changing error message to
```
capabilities of alias should be match or subset of parent bidder
```

Reply:
Error message are updated as part of [96e960b](https://github.com/prebid/prebid-server/pull/2996/commits/96e960b0bdee0e8670ee19308e4f2c42104939be)